string pcd_file_name
---
int32 num_features
int32 feature_size
float64[] shot_feature

int32 num_bins
float64[] color_hist
